server/package.json

```diff
@@ -8,23 +8,23 @@
     "start": "node dist/index.js",
     "start_tsnode": "ts-node src/index.ts",
     "format": "prettier --write .",
-    "dev_tsnode": "nodemon --exec ts-node src/index.ts"
-  },
-  "devDependencies": {
-    "@types/express": "^4.17.13",
-    "@types/lodash": "^4.14.182",
-    "@types/node": "^17.0.10",
-    "nodemon": "^2.0.15",
-    "prettier": "2.6.2",
-    "typescript": "^4.5.5"
+    "dev_tsnode": "nodemon --exec ts-node src/index.ts",
+    "build": "tsc"
   },
+  "devDependencies": {},
   "dependencies": {
     "cors": "^2.8.5",
     "express": "^4.17.2",
     "lodash": "^4.17.21",
     "node-json-db": "^1.5.0",
     "socket.io": "^4.5.0",
-    "ts-node": "^10.4.0"
+    "ts-node": "^10.4.0",
+    "@types/express": "^4.17.13",
+    "@types/lodash": "^4.14.182",
+    "@types/node": "^17.0.10",
+    "nodemon": "^2.0.15",
+    "prettier": "2.6.2",
+    "typescript": "^4.5.5"
```

Comment:
why?

Reply:
heroku is stupid and ignores all dev dependencies, we need tsc and types for build.